prediction/src/preprocess/preprocess_ct.py

```diff
@@ -1,33 +1,40 @@
+import warnings
+
 import numpy as np
 import scipy.ndimage
 from src.preprocess import load_ct
 
 
 class Params:
-    """Params for CT data preprocessing.
+    """Params for CT data pre-processing.
```

Comment:
Please try and make these sorts of changes in a separate commit in future.. much easier to review  :)

Reply:
Ok, thank you for the review :)